integrations/weights-and-bias-tracer.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: integration
+name: Weights & Biases Weave Tracer
```

Comment:
Does W&B have a different product or offering? If not, the name can be simply `Weights & Biases`

Reply:
They have other products, this is just one